package.json

```diff
@@ -92,6 +91,8 @@
     "prettier-cli": "^0.1.0",
     "prettier-eslint": "^8.8.2",
     "prettier-eslint-cli": "^4.7.1",
+    "sinon": "^7.2.7",
```

Comment:
Do you need both?

Reply:
Yes, I need both. For instance I need sinon-chai so I can have access to do the following: expect(res.status).to.have.been.calledWith(500); without it, calling calledWith will throw an error